drake/systems/Simulation.h

```diff
@@ -74,11 +111,24 @@ void simulate(const System& sys, double ti, double tf,
       Eigen::VectorXd::Zero(getNumInputs(sys)));
   typename System::template OutputVector<double> y;
 
+  bool rt_warning_printed = false;
+
   // Take steps from ti to tf.
   double t = ti;
   while (t < tf) {
-    handle_realtime_factor(start, t, options.realtime_factor,
-                           options.timeout_seconds);
+    try {
+      handle_realtime_factor(start, t, options.realtime_factor,
+                             options.timeout_seconds);
+    } catch (std::runtime_error err) {
```

Comment:
I'll address this by modifying `handle_realtime_factor()` to return a bool indicating whether it successfully handled the realtime factor and throwing an exception or printing a warning from `simulate()` if it did not. This should be OK since `simulate()` is the only method that calls `handle_realtime_factor()` in Drake.